.github/workflows/ci.yml

```diff
@@ -142,10 +143,13 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Setup | QEMU
+        uses: docker/setup-qemu-action@v2
+
       - name: Setup | Docker
         uses: docker/setup-buildx-action@v2
         with:
-          platforms: linux/amd64,linux/arm64
+          platforms: linux/amd64,linux/arm64,linux/arm/v7,linux/arm/v6
```

Comment:
Do you actually have a need for these architectures?

Reply:
they are for raspberry pi 1/2/3 with non 64bit archs - mainly for DIY gateways but we have some hobbyists using some of our codebase for this purpose for their own DIY gateways

we are also using armv6 (pi zero) in our light hotspot